css/settings.scss

```diff
@@ -76,6 +76,8 @@ $layout-max-width: 1080px;
  * SPACES && SIZES
 */
 // $space-1: 8px;
+$nav-height: 60px;
```

Comment:
I moved them here because I need them in more than one file at the same time. Otherwise I would put them in their specified file. The submenu needs to know the height of their parent (nav bar o tabs in those cases) to style it properly, that's why. Let me know if you something better.